tools/openms/OpenSwathWorkflow.xml

```diff
@@ -1,598 +1,480 @@
+      <param name="enable_uis_scoring" argument="-enable_uis_scoring" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="false" label="Enable additional scoring of identification assays" help=""/>
+      <param name="out_chrom_type" type="select" optional="false" label="File type of output out_chrom (Also output all computed chromatograms output in mzML (chrom.mzML) or sqMass (SQLite format))">
+        <option value="mzML">mzml</option>
+        <option value="sqMass">sqmass</option>
+      </param>
+      <param type="boolean" name="out_chrom_FLAG" checked="false" truevalue="true" falsevalue="false" label="Generate output out_chrom (Also output all computed chromatograms output in mzML (chrom.mzML) or sqMass (SQLite format))" help=""/>
+      <param type="boolean" name="out_qc_FLAG" checked="false" truevalue="true" falsevalue="false" label="Generate output out_qc (Optional QC meta data (charge distribution in MS1))" help="Only works with mzML input files"/>
+      <param name="min_upper_edge_dist" argument="-min_upper_edge_dist" type="float" optional="true" value="0.0" label="Minimal distance to the upper edge of a Swath window to still consider a precursor, in Thomson" help=""/>
+      <param name="extra_rt_extraction_window" argument="-extra_rt_extraction_window" type="float" optional="true" min="0.0" value="0.0" label="Output an XIC with a RT-window by this much large" help="(e.g. to visually inspect a larger area of the chromatogram)"/>
+      <param name="mz_extraction_window_unit" argument="-mz_extraction_window_unit" display="radio" type="select" optional="false" label="Unit for mz extraction" help="">
+        <option value="Th" selected="true">Th</option>
+        <option value="ppm">ppm</option>
+        <expand macro="list_string_san"/>
+      </param>
+      <param name="mz_extraction_window_ms1_unit" argument="-mz_extraction_window_ms1_unit" display="radio" type="select" optional="false" label="Unit of the MS1 m/z extraction window" help="">
+        <option value="ppm">ppm</option>
+        <option value="Th" selected="true">Th</option>
+        <expand macro="list_string_san"/>
+      </param>
+      <param name="use_ms1_ion_mobility" argument="-use_ms1_ion_mobility" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="true" label="Also perform precursor extraction using the same ion mobility window as for fragment ion extraction" help=""/>
+      <param name="matching_window_only" argument="-matching_window_only" type="boolean" optional="true" truevalue="true" falsevalue="false" checked="false" label="Assume the input data is targeted / PRM-like data with potentially overlapping DIA windows" help="Will only attempt to extract each assay from the *best* matching DIA window (instead of all matching windows)"/>
```

Comment:
`matching_window_only` and `use_ms1_ion_mobility` do not seems to be present in the command section.

Reply:
Indeed you will find none of the parameters (except those for input/output files) in the command, but they find their way directly to the ini file via the json-config file. The magic in here and the following lines:

https://github.com/galaxyproteomics/tools-galaxyp/blob/2a2f239afd08a49d08d09e0d47bd57768777f24c/tools/openms/OpenSwathWorkflow.xml#L66

the agruments as json are due to this:

https://github.com/galaxyproteomics/tools-galaxyp/blob/2a2f239afd08a49d08d09e0d47bd57768777f24c/tools/openms/OpenSwathWorkflow.xml#L148

1. the binary (in this case OpenSwathWorkflow) is used to write a ctd file (with defaults).
2. then `fill_ctd.py` takes the parameters of the tool as json and fills them into the ctd file.
3, the binary is executed with the filled ctd file.